app/main.py

```diff
@@ -2,5 +2,18 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    global_memmory = {}
```

Comment:
There is a typo in the variable name `global_memmory`. It should be `global_memory` to maintain consistency and avoid confusion.